packages/gamut/src/Form/SelectDropdown.tsx

```diff
@@ -0,0 +1,155 @@
+import { css } from '@emotion/react';
+import { each, isArray, isObject } from 'lodash';
+import React, { ChangeEvent, useState } from 'react';
+import ReactSelect, {
+  components,
+  NamedProps,
+  StylesConfig,
+} from 'react-select';
+
+import { SelectIcon, SelectWrapperBaseProps } from './Select';
+import {
+  colorStates,
+  conditionalBorderStyles,
+  conditionalStyleProps,
+  formDropdownStyles,
+  formFieldStyles,
+} from './styles/shared';
+
+type OptionsType = Array<{
+  label: string;
+  value: string | number;
+}>;
+
+type SelectDropdownProps = NamedProps & SelectWrapperBaseProps;
+
+const selectBaseStyles = ({
+  error,
+  activated,
+  isFocused,
+  isDisabled,
+}: conditionalStyleProps) => css`
+  ${formFieldStyles}
+  ${conditionalBorderStyles({ error, activated, isFocused, isDisabled })}
+  line-height: ${theme.lineHeight['base']};
+  display: flex;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+`;
+
+const errorColorState = (error: boolean) => {
+  const color = error ? colorStates.error.color : colorStates.base.color;
+  return color;
+};
+
+const customStyles: StylesConfig<OptionsType, false> = {
+  dropdownIndicator: (provided, state) => ({
+    color: errorColorState(state.selectProps.error),
+    display: 'flex',
+    padding: '0',
+  }),
+
+  container: (provided) => ({
+    ...provided,
+    pointerEvents: 'visible',
+    width: '100%',
+    minWidth: '7rem',
+  }),
+
+  menu: (provided) => ({
+    ...provided,
+    ...formDropdownStyles,
+  }),
+
+  option: (provided, state) => ({
+    padding: '14px 11px 14px 11px',
+    cursor: 'pointer',
+    backgroundColor:
+      (state.isSelected && colorStates.dropdown.selected.backgroundColor) ||
+      (state.isFocused && colorStates.dropdown.focused.backgroundColor) ||
+      'transparent',
+    '&:hover': {
+      backgroundColor: colorStates.dropdown.focused.backgroundColor,
+    },
+  }),
+
+  control: (provided, state) => ({
+    ...selectBaseStyles({
+      error: state.selectProps.error,
+      activated: state.selectProps.activated,
+      isFocused: state.isFocused,
+      isDisabled: state.isDisabled,
+    }),
+  }),
+
+  singleValue: (provided, state) => ({
+    color: errorColorState(state.selectProps.error),
+  }),
+
+  valueContainer: (provided) => ({
+    ...provided,
+    padding: 0,
+  }),
+};
+
+const DropdownIndicator = (
+  props: ElementConfig<typeof components.DropdownIndicator>
+) => {
+  return (
+    <components.DropdownIndicator {...props}>
+      <SelectIcon size={16} />
+    </components.DropdownIndicator>
+  );
+};
+
+export const SelectDropdown: React.FC<SelectDropdownProps> = ({
+  defaultValue,
+  options,
+  error,
+  id,
+  disabled,
+  ...rest
+}) => {
+  const [activated, setActivated] = useState(false);
+
+  const changeHandler = (event: ChangeEvent<HTMLSelectElement>) => {
+    rest?.onChange?.(event);
+    setActivated(true);
+  };
+
+  const selectOptions: OptionsType = [];
+
+  if (isArray(options)) {
```

Comment:
Nit: I think it's a little cleaner to use native operations when possible, instead of pulling in extraneous code:

```suggestion
  if (options instanceof Array) {
```

...and later down, can `isObject` be replaced with `if (options)`, since the only options are array, object, and undefined?